App/app/android/app/src/debug/AndroidManifest.xml

```diff
@@ -4,5 +4,6 @@
          the Flutter tool needs it to communicate with the running application
          to allow setting breakpoints, to provide hot reload, etc.
     -->
-    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
 </manifest>
```

Comment:
Please remove these changes, as it is not required to access the network state.

Reply:
Won't do